source/miscellaneous.rst

```diff
@@ -1,74 +1,62 @@
+    .. figure:: _static/images/jar-doc-4.png
+        :align: center
+        
+        This will run the ``Main`` file in the ``sir\textunderscore ct`` folder
 
 5. The sbt shell will display that the following tests were *unsuccessful*:-
+   
         a. item Neo4jProviderTest
         b. item BatchNeo4jProviderTest
         c. item ReadOperationsStreamTest
         d. item WriteOperationsStreamTest
-Go to ``BharatSim\src\test\scala\com\bharatsim\engine\graph\neo4j`` and comment out (``ctrl + /``) all the lines in the test files for tests (a) and (b). Open the ``queryBatching`` folder in the same directory and comment out all lines for tests (c) and (d). 
 
-7. Open ``BharatSim\target\scala-2.13`` where you will find the new JDK JAR file created. It should have the name``engine-assembly-0.1``. You can now keep the file here or move it to a folder/directory of your choice.
 
-8. Open a shell(powershell/gitbash/linux shell , etc.) in the directory/folder in which you have saved your JDK JAR file and type ``java -jar jarfilename``.
+Go to ``BharatSim\src\test\scala\com\bharatsim\engine\graph\neo4j`` and comment out (``ctrl + /``) all the lines in the test files for tests (a) and (b). Open the ``queryBatching`` folder in the same directory and comment out all lines for tests (c) and (d). 
 
-.. figure:: _static/images/jar-doc-3.png
-    :width: 200px
-    :align: center
-    :height: 100px
-    :alt: alternate text
-    :figclass: align-center
-    
-.. class:: center
+1. Open ``BharatSim\target\scala-2.13`` where you will find the new JDK JAR file created. It should have the name``engine-assembly-0.1``. You can now keep the file here or move it to a folder/directory of your choice.
 
-    GIT bash
+2. Open a terminal (powershell/gitbash/linux shell, etc.) in the directory/folder in which you have saved your JDK JAR file and type ``java -jar jarfilename``.
+
+    .. figure:: _static/images/jar-doc-3.png
+        :align: center
 
 Using args in main method
-==========================
-While using the ``run`` command on the sbt shell, one can pass in some ``string`` arguments. These arguements can be called by the main function. 
+-------------------------
+
+While using the ``run`` command on the sbt shell, one can pass in some ``string`` arguments. These arguements can be called by the main function.
+
 For example, one might want to change the name of the output file everytime they run the code. Instead of changing the output name manually each time, one can write a code like the one given below.
 
-.. code::
+.. code-block:: scala
 
       def main(args: Array[String]): Unit = {
         outputName = args(0)
         }
         
 One could use the ``outputName`` and use it in the name of the csv file, for eg. 
 
-.. code::
+.. code-block:: scala
 
     SimulationListenerRegistry.register(
         new CsvOutputGenerator("C:\\Users\\username\\Documents\\"+outputName+".csv", new SEIROutputSpec(context))
```

Comment:
Can you change this link, please? I know you didn't write it, but still. Just make it ``outputName+".csv"``, the rest of it is unnecessary. In general, I'd like all our paths to be from the root `BharatSim` folder. i.e. just `src/` or `target/` etc. with the forward slash.

Reply:
(But don't worry about the slashes and other paths if it's too hard, @prathithbhargav and @SoumilK1 will have to make those changes.)

Reply:
I did something in 7226e9a. Is that what the path is supposed to be here? Just a bit confused.